doc/release-notes.md

```diff
@@ -33,6 +33,25 @@ frequently tested on them.
 Notable changes
 ===============
 
+Performance Improvements
+--------------
+
+Validation speed and network propogation performance have been greatly
+improved, leading to much shorter sync and initial block download times.
+
+- The script signature cache has been reimplemented as a high-performance,
+  low-overhead "cuckoo cache"
+- Assumed-valid blocks have been introduced which allows script validation to
+  be skipped for ancestors of known-good blocks, without changing the security
+  model. See below for more details.
+- In some cases, compact blocks are now relayed before being fully validated as
+  per BIP152.
```

Comment:
I'm not sure what you're getting at here, sounds the same to me :)
Please suggest a complete alternative.